chapters/crud.md

```diff
@@ -169,7 +169,7 @@ Unfortunately there are some limitations to what Jina can do for the moment. The
 
 5. **Compound Indexer**
 
-While using a `!CompoundIndexer`, it is not possible to return embeddings to the next Driver in the Flow, by virtue of its design. By definition, the [!CompoundIndexer](https://github.com/jina-ai/jina/blob/master/jina/resources/executors.requests.CompoundIndexer.yml) has an `!ExcludeQL` for all the embeddings:
+While using a `!CompoundIndexer`, it is not possible to return embeddings to the next Driver in the Flow. This is because  the [!CompoundIndexer](https://github.com/jina-ai/jina/blob/master/jina/resources/executors.requests.CompoundIndexer.yml) has an `!ExcludeQL` for all the embeddings during `IndexRequest` and `UpdateRequest`
```

Comment:
Since it's excluded for IndexRequest, UpdateRequest and SearchRequest, can we have embedding for other requests like DeleteRequest?